openedx/core/djangoapps/xblock/runtime/blockstore_runtime.py

```diff
@@ -133,7 +133,9 @@ def save_block(self, block):
             if not learning_context.can_edit_block(self.user, block.scope_ids.usage_id):
                 log.warning("User %s does not have permission to edit %s", self.user.username, block.scope_ids.usage_id)
                 raise RuntimeError("You do not have permission to edit this XBlock")
-        olx_str, static_files = serialize_xblock(block)
+        serialized = serialize_modulestore_block_for_blockstore(block)
```

Comment:
I guess we should rename this function to just `serialize_block_for_blockstore` since we're already passing in a blockstore block here.

Reply:
it is called in a bunch of other places.